packages/protocol/contracts/common/UsingPrecompiles.sol

```diff
@@ -2,7 +2,16 @@ pragma solidity ^0.5.3;
 
 // TODO(asa): Limit assembly usage by using X.staticcall instead.
 contract UsingPrecompiles {
+  address constant TRANSFER = address(0xff - 2);
```

Comment:
FYI https://github.com/celo-org/celo-monorepo/pull/2324

Reply:
Nice. Seems the mload from #2324 may be OK.